CodeNow/runnable-angular
# SAN-1398 Fix container save bugs

We have a bug where if you try to change an env var on a container that is currently building you get an error. This fixes that.

To test:
1. Use dockerfile:

```
FROM ubuntu
RUN sleep 60
CMD sleep 999999
```
1. Build
2. Within 60 seconds change an env var
3. Save
4. Notice it triggers a rebuild instead of a redeploy.
